### PR TITLE
[dev-v2.8] Create rancher-provisioning-capi 103.2.0+up0.0.1

### PR DESCRIPTION
**Pull request by Oats87**
## Problem
rancher-provisioning-capi `103.1.0` contained a bumped version of `cluster-api` that was incompatible with Rancher `v2.8.0` - `v2.8.2`

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -17448,6 +17448,32 @@ entries:
...
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Rancher Provisioning CAPI Controller Manager
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/kube-version: '>=1.23.0-0'
+      catalog.cattle.io/namespace: cattle-provisioning-capi-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/provides-gvr: apps.deployment/v1
+      catalog.cattle.io/rancher-version: '>= 2.7.0-0'
```
Comment (lucasmlp) [inline index.yaml]: This chart should not be compatible with `Rancher v2.7`

**Review by lucasmlp — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Could you add `103.2.0+up0.0.1` to the `release.yaml` ?

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Review by jiaqiluo — APPROVED** [review]
LGTM. 
```
>  diff -r 103.0.0+up0.0.1 103.2.0+up0.0.1
diff --color -r 103.0.0+up0.0.1/Chart.yaml 103.2.0+up0.0.1/Chart.yaml
10c10
<   catalog.cattle.io/rancher-version: '>= 2.7.0-0'
---
>   catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
22c22
< version: 103.0.0+up0.0.1
---
> version: 103.2.0+up0.0.1
```

**Pull request merged**